app/views/customers/_subscriptions.html.erb

```diff
@@ -1 +1,27 @@
-<h1 style="color:black;">subscriptions partial</h1>
+<div style="color:black;">
+  <div>
+    
+  </div>
+  <% if @customer.total_count != 0 %>
+    <table class="bordered">
+      <thead>
+        <tr>
+          <td>Product</td>
+          <td>Price</td>
+          <td>Last Payment</td>
+        </tr>
+      </thead>
+      <tbody>
+      <% @customer.data.each do |subscription| %>
```

Comment:
Maybe in the controller we could drop the `.subscriptions` and here we could loop through `@customer.subscribtions.data.each`? It might be useful as we add onto the customer dashboard to have all the customer data in the instance variable and not just the subscription.